metabase/metabase
# Make using the QA DBs easier in local development

## Description

Adds a docker-compose file to spin up all the QA dbs together, and adds some handy-dandy yarn scripts to not have to fuss with the details of spinning up containers and environment variables.

This will be helpful as these external DBs will be needed in more of our tests as we start testing actions.

## How to test

- pull this branch
- have docker running
- run `yarn test-cypress-open-qa`
- this should spin up all the qa DBs and open cypress, and you should see all the snapshots create correctly during startup:

![Screen Shot 2023-01-26 at 10 51 40 AM](https://user-images.githubusercontent.com/30528226/214911550-f95807f0-d6ff-46dc-8a0a-e9c50615b592.png)
